src/zenml/models/v2/core/step_run.py

```diff
@@ -156,6 +160,10 @@ class StepRunResponseBody(WorkspaceScopedResponseBody):
         title="The input artifact versions of the step run.",
         default_factory=dict,
     )
+    input_types: Dict[str, StepRunInputArtifactType] = Field(
```

Comment:
I'm wondering whether we can just include that in the `inputs` somehow? Maybe like this?

```python
class StepRunInputResponse(ArtifactVersionResponse):
  input_type: StepRunInputArtifactType
```

Reply:
reworked as suggested